i18n/en.json

```diff
@@ -939,7 +939,7 @@
   "admin.privacy.showFullNameDescription": "When false, hides the full name of members from everyone except System Administrators. Username is shown in place of full name.",
   "admin.privacy.showFullNameTitle": "Show Full Name: ",
   "admin.purge.button": "Purge All Caches",
-  "admin.purge.purgeDescription": "This will purge all the in-memory caches for things like sessions, accounts, channels, etc. Deployments using High Availability will attempt to purge all the servers in the cluster.  Purging the caches may adversly impact performance.",
+  "admin.purge.purgeDescription": "This will purge all the in-memory caches for things like sessions, accounts, channels, etc. Deployments using High Availability will attempt to purge all the servers in the cluster.  Purging the caches may adversely impact performance.",
```

Comment:
Is the double white space before "Purging" intended?

Reply:
Good catch. Removed, I don't see why it would have been intentional.